docs/developer/eventing/sinks/README.md

```diff
@@ -1,29 +1,12 @@
 # About sinks
 
-When you create an event source, you can specify a _sink_ where events are sent to from the source. A sink is an Addressable resource that can receive incoming events from other resources. Knative Services, Channels, and Brokers are all examples of sinks.
+When you create an event source, you can specify a _sink_ where events are sent to from the source. A sink is an _Addressable_ or a _Callable_ resource that can receive incoming events from other resources. Knative Services, Channels, and Brokers are all examples of sinks.
 
-You can also connect a Trigger to a sink, so that events are filtered before they are sent to the sink. A sink that is connected to a Trigger is configured as a `subscriber` in the Trigger resource spec.
+Addressable objects
+:   Addressable objects receive and acknowledge an event delivered over HTTP to an address defined in their `status.address.url` field. As a special case, the core [Kubernetes Service object](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#service-v1-core) also fulfils the Addressable interface.
 
-For example:
-
-```yaml
-apiVersion: eventing.knative.dev/v1
-kind: Trigger
-metadata:
-  name: <trigger-name>
-spec:
-...
-  subscriber:
-    ref:
-      apiVersion: eventing.knative.dev/v1alpha1
-      kind: KafkaSink
-      name: <kafka-sink-name>
-```
-
-Where;
-
-- `<trigger-name>` is the name of the Trigger being connected to the sink.
-- `<kafka-sink-name>` is the name of a KafkaSink object.
+Callable objects
+:   Callable objects are able to receive an event delivered over HTTP and transform the event, returning 0 or 1 new events in the HTTP response. These returned events may be further processed in the same way that events from an external event source are processed.
```

Comment:
You want to either mention `status.address.url`, or mention that is an _Addressable_.